app/services/invoices/refresh_draft_service.rb

```diff
@@ -31,6 +31,7 @@ def call
 
       ActiveRecord::Base.transaction do
         invoice.update!(ready_to_be_refreshed: false) if invoice.ready_to_be_refreshed?
+        invoice.customer.flag_wallets_for_refresh
```

Comment:
This will be executed every 5 minutes for every draft invoice. Is there a simple way to only flag them if something has actually been updated?